playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -156,6 +159,7 @@
   tags:
     - install
     - update_airship_osh_site
+    - add_computenode
 
 - name: Wait 10 minutes before checking the action status
```

Comment:
what's your observation of the time needed for add compute redeploy osh,? If it is less than 10 minutes, can we put in logic that we don't wait for 10 minutes when add compute? or a shorter time?

Reply:
> what's your observation of the time needed for add compute redeploy osh,? If it is less than 10 minutes, can we put in logic that we don't wait for 10 minutes when add compute? or a shorter time?

I am skipping wait for 10 mins ?  because I am directly doing update software .  you can see tags are for some of the tasks.

Reply:
you are right...